src/style.css

```diff
@@ -194,3 +194,7 @@ body {
   color: rgba(255, 255, 255, 0.9);
   text-align: center;
 }
+
+.icon {
+  color: rgba(218, 217, 216, 0.863) !important;
```

Comment:
did you need important? can't it be replaced and reorganised?

Reply:
Unfortunately, Semantic is hard to override :(/